src/main/kotlin/me/obsilabor/layercontrol/config/ClothConfigManager.kt

```diff
@@ -51,6 +51,15 @@ object ClothConfigManager {
             }
             .setDefaultValue(LayerControlConfig.DEFAULT.parrotVariantRight)
             .build())
+        general.addEntry(entryBuilder.startFloatField(Component.translatable("layercontrol.option.bigHeadSize"), config?.bigHeadScale ?: 2f)
+            .setSaveConsumer {
+                if (it > 100f)
```

Comment:
Could you add a minimum limit for 1.0? Because the custom head would be rendered inside of the player head if its below 1.0